index.html

```diff
@@ -594,6 +597,23 @@ <h3>
           doPaymentRequest();
         </pre>
       </section>
+      <section>
+        <h2>
+          Using with cross-origin iframes
+        </h2>
+        <p>
+          To indicate that a cross-origin [^iframe^] is allowed to invoke the
+          payment request API, the [^iframe/allow^] attribute along with the
+          "payment" keyword can be specified on the [^iframe^] element.
+        </p>
+        <pre class="example html" title=
+        "Using Payment Request API with cross-origin iframes">
+            &lt;iframe
+              src="https://cross-origing.example"
+              allow="payment"&gt;
```

Comment:
It was pointed out to me on https://bugs.chromium.org/p/chromium/issues/detail?id=1127988#c2 that the equivalent of `allowpaymentrequest` is actually `allow="payment *"`.

If I'm reading the [Permissions Policy spec][1] correctly, `allow="payment"` (without the *) only enables Payment Request API in same-origin iframes. @clelland to confirm.

[1]: https://w3c.github.io/webappsec-permissions-policy/#allowlists

Reply:
`allow="payment *"` is the right replacement, yes. +1 to that change.

`allow="payment"` is shorthand for `allow="payment 'src'"`, which means roughly that the origin in the `src` attribute is granted permission, but if the iframe navigates away from that origin, it will be denied.

Reply:
ok, good catch. I might add both to the example, just to make sure it's clear which to use in situations where the iframe will be navigated to multiple origins. I think `allow="payment"` should be preferred in most cases, as `allow="payment *"` might be overly generous.